bundle/src/main/java/com/adobe/acs/commons/wcm/impl/SiteMapServlet.java

```diff
@@ -78,14 +79,22 @@
             description = "If true, the last modified value will be included in the sitemap.")
     private static final String PROP_INCLUDE_LAST_MODIFIED = "include.lastmod";
 
-    @Property(label = "Change Frequency Properties",
+    @Property(label = "Change Frequency Properties", unbounded = PropertyUnbounded.ARRAY,
             description = "The set of JCR property names which will contain the change frequency value.")
     private static final String PROP_CHANGE_FREQUENCY_PROPERTIES = "changefreq.properties";
 
-    @Property(label = "Priority Properties",
+    @Property(label = "Priority Properties", unbounded = PropertyUnbounded.ARRAY,
             description = "The set of JCR property names which will contain the priority value.")
     private static final String PROP_PRIORITY_PROPERTIES = "priority.properties";
 
+    @Property(label = "DAM Folder Property",
+            description = "The JCR property name which will contain DAM folders to include in the sitemap.")
+    private static final String PROP_DAM_ASSETS_PROPERTY = "damassets.property";
+
+    @Property(label = "DAM Asset MIME Types", unbounded = PropertyUnbounded.ARRAY,
+            description = "MIME types allowed for DAM assets.")
+    private static final String PROP_DAM_ASSETS_TYPES = "damassets.types";
```

Comment:
Good question on whether accepted MIME types should be configured at the page level or the servlet level.  I think for a first cut configuration at the servlet level is a good start for simplicity sake.  If we allow filtering MIME types at the folder level, then we'd have to handle things like a parent folder configured one way with a child folder configured a different way.  If someone needs that level of functionality we could update this later in a backward-compatible way by making the OSGI configuration the default and allowing overrides at the page level.